demos/fab.html

```diff
@@ -93,52 +90,84 @@
       </button>
 
       <section>
-        <fieldset>
-          <legend>Normal FABs</legend>
-          <button class="mdc-fab material-icons" aria-label="Favorite">
-            <span class="mdc-fab__icon">
-              favorite_border
-            </span>
-          </button>
-          <button class="mdc-fab mdc-fab--mini material-icons" aria-label="Favorite">
-            <span class="mdc-fab__icon">
-              favorite_border
-            </span>
-          </button>
-          <button class="mdc-fab mdc-fab--plain material-icons" aria-label="Favorite">
-            <span class="mdc-fab__icon">
-              favorite_border
-            </span>
-          </button>
-          <button class="mdc-fab mdc-fab--mini mdc-fab--plain material-icons" aria-label="Favorite">
-            <span class="mdc-fab__icon">
-              favorite_border
-            </span>
-          </button>
-        </fieldset>
-        <fieldset>
-          <legend>CSS Only FABs</legend>
-          <button class="mdc-fab material-icons" aria-label="Favorite" data-demo-no-js>
-            <span class="mdc-fab__icon">
-              favorite_border
-            </span>
-          </button>
-          <button class="mdc-fab mdc-fab--mini material-icons" aria-label="Favorite" data-demo-no-js>
-            <span class="mdc-fab__icon">
-              favorite_border
-            </span>
-          </button>
-          <button class="mdc-fab mdc-fab--plain material-icons" aria-label="Favorite" data-demo-no-js>
-            <span class="mdc-fab__icon">
-              favorite_border
-            </span>
-          </button>
-          <button class="mdc-fab mdc-fab--mini mdc-fab--plain material-icons" aria-label="Favorite" data-demo-no-js>
-            <span class="mdc-fab__icon">
-              favorite_border
-            </span>
-          </button>
-        </fieldset>
+        <legend>FABs with Ripple</legend>
+        <div class="demo-fabs">
+          <figure>
+            <button class="mdc-fab material-icons" aria-label="Favorite">
+              <span class="mdc-fab__icon">
+                favorite_border
+              </span>
+            </button>
+            <figcaption>
+              <div>Themed FAM</div>
+            </figcaption>
+          </figure>
+          <figure>
+            <button class="mdc-fab mdc-fab--mini material-icons" aria-label="Favorite">
+              <span class="mdc-fab__icon">
+                favorite_border
+              </span>
+            </button>
+            <figcaption>
```

Comment:
Was your thought that including these captions on the first set of buttons is sufficient? (Just noting that they're only present on the ripple set, not the CSS-only set.)

Reply:
Yes, I think it is sufficient. (And whenever I copy paste stuff across sections I run the risk of forgetting to update it in a later PR.)

Reply:
Yup, can definitely agree with that logic and figured that might be part of the reason :)